src/lib_c/x86_64-windows-msvc/c/basetsd.cr

```diff
@@ -4,4 +4,5 @@ lib LibC
   {% else %}
     alias ULONG_PTR = ULong
   {% end %}
+  alias PULONG_PTR = ULONG_PTR*
```

Comment:
Don't make any of the `Pxxx` aliases, just use `xxx*`. It makes it much clearer whats going on.